contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1954,40 +1977,43 @@ class tsqlBuilder : public tsqlCommonMutator
 
 	void exitChar_string(TSqlParser::Char_stringContext *ctx) override
 	{
-		if (ctx->STRING())
-		{
-			std::string str = ctx->STRING()->getSymbol()->getText();
+		std::string str = getFullText(ctx);			
```

Comment:
Would you point out in which condition it'll be something different other than ctx->STRING() ?

Reply:
The test for STRING was redundant given the grammar rule. It cannot be anything other than STRING:
```
char_string
    : STRING // single-quoted string; may also be a double-quoted string in case of SET QUOTED_IDENTIFIER OFF
    ;
```